source/develop/release-management/features/infra/taskmanagerdetailed.html.md

```diff
@@ -127,14 +127,14 @@ The *categories* field will enable filtering jobs by categories. An action type
 ##### Main Task Manager Class Diagram
 
 The following class diagrams describes the entities participating in the the Task Manager feature:
- ![](async-task-main-class-diagram.jpeg "fig:async-task-main-class-diagram.jpeg")
+ ![](/images/wiki/Async-task-main-class-diagram.jpeg)
```

Comment:
This link, and several below, have been changed to have an uppercase first letter. Is that intentional? It doesn't matter as long as this link (and the others similar to it) has the same case as the filename on disk.

Note: Several others already start with an uppercase letter. It's probably a wiki-ism — but it's odd that this is inconsistent.